docker-compose.dev.yml

```diff
@@ -14,14 +14,16 @@ services:
       - POSTGRES_HOST_WRITER=database
       - POSTGRES_PORT=5432
     volumes:
-      - .:/opt/src
+      - ./scripts:/opt/src/scripts
```

Comment:
we only need to `mount` scripts to be able to run tests (source code is already in the docker image)